deploy/README.md

```diff
@@ -253,7 +255,7 @@ spec:
                     operator: Exists
 ```
 
-### Running topolvm-scheduler using Deployment and Service
+#### Running topolvm-scheduler using Deployment and Service
```

Comment:
`OPTIONAL: tune the node scoring` section is level2 heading, but it is only related to topolvm-scheduler.
Would you make it smaller?

Reply:
fixed it.
https://github.com/topolvm/topolvm/blob/17c094b979ecf53030ff1850a8b8f923e7264a9e/deploy/README.md